src/main/java/com/mrbysco/unhealthydying/util/UnhealthyHelper.java

```diff
@@ -30,11 +30,20 @@ public static void changeModifier(PlayerEntity player, double modifierValue) {
             if(attributeInstance != null) {
                 AttributeModifier oldAttribute = attributeInstance.getModifier(Reference.HEALTH_MODIFIER_ID);
                 if(oldAttribute != null) {
-                    if(oldAttribute.getAmount() != modifierValue)
-                        HealthUtil.sendHealthMessage(player, (int)attributeInstance.getValue(), (int)modifierValue);
-
+                    if(oldAttribute.getAmount() != modifierValue) {
+                        int newModifier = -(20 + (int) modifierValue);
+                        if (oldAttribute.getAmount() < modifierValue)
+                            HealthUtil.sendHealthMessage(player, newModifier, 1);
+                        else
+                            HealthUtil.sendHealthMessage(player, newModifier, -1);
+                    }
                     attributeInstance.removePermanentModifier(Reference.HEALTH_MODIFIER_ID);
                 }
+                else
+                {
+                    int newModifier = -(20 + (int) modifierValue);
```

Comment:
Should be using `attributeInstance.getBaseValue()` in case an external mod changes the base value